react/react.go

```diff
@@ -9,15 +9,30 @@ import (
 )
 
 const (
-	reactProps = "props"
-	reactState = "state"
-
-	nestedProps = "_props"
-	nestedState = "_state"
+	reactCompProps                     = "props"
+	reactCompState                     = "state"
+	reactCompDisplayName               = "displayName"
+	reactCompSetState                  = "setState"
+	reactCompGetInitialState           = "getInitialState"
+	reactCompShouldComponentUpdate     = "shouldComponentUpdate"
+	reactCompComponentDidMount         = "componentDidMount"
+	reactCompComponentWillReceiveProps = "componentWillReceiveProps"
+	reactCompComponentWillMount        = "componentWillMount"
+	reactCompComponentWillUnmount      = "componentWillUnmount"
+	reactCompRender                    = "render"
+
+	reactCreateElement = "createElement"
+	reactCreateClass   = "createClass"
+	reactDomRender     = "render"
```

Comment:
DOM is an acronym for Document Object Model. According to Go style, they should have a consistent case, so this should be:

```
reactDOMRender
```

Source: https://github.com/golang/go/wiki/CodeReviewComments#initialisms.